Examples/Tests/ohm_solver_EM_modes/PICMI_inputs_rz.py

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env python3
+#
+# --- Test script for the kinetic-fluid hybrid model in WarpX wherein ions are
+# --- treated as kinetic particles and electrons as an isothermal, inertialess
+# --- background fluid. The script is set up to produce parallel normal EM modes
+# --- in a metallic cylinder and is run in RZ geometry.
+# --- As a CI test only a small number of steps are taken.
+
+import argparse
+import sys
+
+import dill
+from mpi4py import MPI as mpi
+import numpy as np
+
+from pywarpx import picmi
+
+constants = picmi.constants
+
+comm = mpi.COMM_WORLD
+
+simulation = picmi.Simulation(verbose=0)
+# make a shorthand for simulation.extension since we use it a lot
+sim_ext = simulation.extension
```

Comment:
It looks like the `sim_ext` variable is actually not used in the script ; do you want to remove it?